src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="body">
+    <div class="stopwatch">
+      <div class="innerD"></div>
+      <div class="stopwatch__minutes"></div>
+      <div class="stopwatch__seconds"></div>
```

Comment:
The element for seconds should have a class that follows BEM methodology, such as 'stopwatch__seconds-hand' or 'stopwatch__seconds-display' to clearly describe its purpose.